README.md

```diff
@@ -81,11 +81,21 @@ $ pip install -e ".[dev]"
 $ pre-commit install
 ```
 
+On **Windows**, prebuilt docker images are hosted on our DockerHub. 
```

Comment:
Perhaps "For Windows"? Arguably the availability of Docker images isn't directly related to the host OS?

Reply:
```suggestion
For **Windows**, prebuilt docker images are hosted on our DockerHub. 
```